### PR TITLE
Navbar

### PR DESCRIPTION
**Pull request by ZestyScrubz**
Closes #16 
## Overview
Navbar with working intractable link
Navbar with mobile compatibility

Improve design
- Center links
- Background colour
- Font?

### CODE REVIEW

**Review by Cheekane — DISMISSED** [review]
👍 Good job

**Comment by Lunchii** [comment]
Yes 🧇

**Review by Lunchii — DISMISSED** [review]
😎

File: app/components/NavBar.tsx [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    const handleToggle = () => { 
+        setIsOpen(!isOpen); 
+    };
+
+    return (
+            <div className="w-full h-full bg-transparent">
+                {/* Desktop Menu */}
+                <a href="/home" className="flex-shrink-0 fixed"> 
+                    <img src={LKPC_Logo.src} alt="LKPC Logo" className="w-24 h-24 hover:scale-105 transition-all" />
+                </a>
```
Comment (jerpyk) [inline app/components/NavBar.tsx:20]: nice, clicking logo needs to lead to home page

**Review by jerpyk — APPROVED** [review]
nice

File: app/components/NavBar.tsx (hunk 1 above)
Comment (jerpyk) [reply app/components/NavBar.tsx:20]: nvm it works but is just glitchy

**Review by Cheekane — APPROVED** [review]
Good work

**Pull request merged**